golang/cosmos/scripts/protocgen.sh

```diff
@@ -3,12 +3,12 @@
 set -eo pipefail
 
 protoc_gen_gocosmos() {
-  if ! grep "github.com/gogo/protobuf => github.com/regen-network/protobuf" go.mod &>/dev/null ; then
+  if ! grep "github.com/gogo/protobuf => github.com/regen-network/protobuf" go.mod &> /dev/null; then
```

Comment:
The default of the Prettier plugin seems to be different than the default of the `shfmt` command. I think we could flip it with the `spaceRedirects` option

Reply:
FWIW, I like the space. That's what I would have written manually.

Reply:
same. I got used to the `shfmt` CLI taking it out, but I'll get used to adding the flag to keep it :)